samples/sampler_filter.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+<!-- Ported from the OpenGL Samples Pack https://github.com/g-truc/ogl-samples/blob/master/tests/gl-330-sampler-filter.cpp -->
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - sampler_filter</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="utility.js"></script>
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - sampler_filter</div>
+
+    <script id="vs" type="x-shader/x-vertex">#version 300 es
+        #define POSITION 0
```

Comment:
These GLSL constants are OK, but they should probably be named, for example, `POSITION_LOCATION`, and the JavaScript variable (`vertexPosLocation` in this case) should have the same name.  This will be much easier for developers new to WebGL 2 to understand.

Also open a new pull request to make this consistent across all samples.